sdk/digitaltwins/Azure.DigitalTwins.Core/tests/SimpleNewtonsoftDtModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Newtonsoft.Json;
+
+namespace Azure.DigitalTwins.Core.Tests
+{
+    public class SimpleNewtonsoftDtModel
+    {
+        [JsonProperty("$dtId")]
+        public string Id { get; set; }
+    }
```

Comment:
Any reason not to fill out the model? If we're testing...

Reply:
Honestly I figured that to test the serializer/deserializer, all we need is to make sure we can use the attributes that newtonsoft provides correctly. I didn't want to implement the whole BasicDigitalTwin with Newtonsoft just for the sake of testing. 
If you think having a BasicDigitalTwin implemented with Newtonsoft is necessary, I can add that to the core package and we can publish that as well.

Reply:
Might as well be thorough with testing.

It could serve as an example for others too.

Reply:
As discussed offline, I will implement the basic digital twin with Newtonsoft as a sample code after this PR is merged and the package has been published.